features/step_definitions/logging.rb

```diff
@@ -1056,3 +1056,125 @@
     | pod_label       | #{pod_label}     |
   })
 end
+
+
+
+Given /^I have(?: "(\w+)")? log pod in project #{QUOTED}$/ do | log_type, project_name |
+    log_type ||= "any"
+    file_dir = "#{BushSlicer::HOME}/testdata/logging/loggen"
+
+    unless project(project_name).exists?
+        step %Q/I run the :new_project client command with:/,table(%{
+          | project_name | #{project_name} |
+        })
+        raise "Error creating namespace" unless @result[:success]
+    end
+
+    case log_type
+      when "json"
+        template_file = "#{file_dir}/container_json_log_template.json"
+      when "flat"
```

Comment:
@anpingli  what's the difference between `flat` and `unicode`? From the file name, I see now difference.

Reply:
If you need to declare these 2 types, then you can simplify it to:
```
      when "unicode", "flat"
        template_file = "#{file_dir}/container_json_unicode_log_template.json"
```